dotCMS/src/main/java/com/dotcms/vanityurl/business/VanityUrlAPIImpl.java

```diff
@@ -65,17 +67,24 @@ public VanityUrlAPIImpl(final ContentletAPI contentletAPI,
 
     @Override
     public List<VanityUrl> getActiveVanityUrls(final User user) {
-        return searchAndPopulate(GET_ACTIVE_VANITY_URL, user);
+        return searchAndPopulate(GET_ACTIVE_VANITY_URL, user, null, null);
     }
 
     @Override
     public List<VanityUrl> getActiveVanityUrlsBySiteAndLanguage(final String siteId,
                                                                 final long languageId, final User user) {
 
-        final String luceneQuery = GET_ACTIVE_VANITY_URL + " +conHost:" + siteId
+        String HOST_QUERY;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)